go/vt/vttablet/tabletserver/health_streamer.go

```diff
@@ -146,25 +143,27 @@ func (hs *healthStreamer) Open() {
 	if hs.conns != nil {
 		// if we don't have a live conns object, it means we are not configured to signal when the schema changes
 		hs.conns.Open(hs.dbConfig, hs.dbConfig, hs.dbConfig)
-		hs.ticks.Start(func() {
-			if err := hs.reload(); err != nil {
-				log.Errorf("periodic schema reload failed in health stream: %v", err)
-			}
-		})
-
 	}
+}
 
+// startSchemaNotifications registers the healthStreamer to be notified by the schema engine when some schema change occurs.
+func (hs *healthStreamer) startSchemaNotifications() {
```

Comment:
nit: can we call this as `subscribeSchemaNotification` or something like that?

Reply:
I removed this function entirely.